### PR TITLE
add two beginner projects

### PR DESCRIPTION
**Pull request by dansholds**
Added two projects:

Log Archiving tool for DevOps & Linux Roadmaps.
Basic Dockerfile for DevOps & Docker Roadmaps.

### CODE REVIEW

File: src/data/projects/basic-dockerfile.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+difficulty: 'beginner'
+nature: 'CLI'
+skills:
+  - 'docker'
+  - 'dockerfile'
+  - 'linux'
+  - 'devops'
+seo:
+  title: 'Basic Dockerfile'
+  description: 'Build a basic Dockerfile to create a Docker image.'
```
Comment (dansholds) [inline src/data/projects/basic-dockerfile.md:15]:
```suggestion
  description: 'Write a basic Dockerfile to create a Docker image.'
```

File: src/data/projects/log-archive-tool.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+title: 'Log Archive Tool'
+description: 'Build a tool to archive logs from the CLI.'
+isNew: false
+sort: 1
```
Comment (dansholds) [inline src/data/projects/log-archive-tool.md:5]:
```suggestion
sort: 2
```

**Comment by dansholds** [comment]
@kamranahmedse what happens if the sort is = to 2 on a roadmapId that only has 1 project?

**Comment by kamranahmedse** [comment]
@dansholds, it won't matter. It just sorts the projects in increasing order, and even if some sorting numbers are missing, it should be fine.

**Pull request merged**